src/dash-table/derived/cell/eventHandler.ts

```diff
@@ -18,34 +16,36 @@ export enum Handler {
 export type CacheFn = (...args: CacheArgs) => Function;
 export type HandlerFn = (...args: any[]) => any;
 
-const getter = (propsFn: () => ICellFactoryProps): CacheFn => {
-    const cache = memoizerCache<CacheArgs, GetterArgs, Function>((...args: GetterArgs) => {
-        let [
-            handler,
-            rowIndex,
-            columnIndex
-        ] = args;
+export default (propsFn: () => ICellFactoryProps) => new EventHandler(propsFn).get;
 
-        return handler && handler.bind(undefined, rowIndex, columnIndex);
-    });
+class EventHandler {
+    constructor(private readonly propsFn: () => ICellFactoryProps) {
+
+    }
 
-    const handlers = new Map<Handler, HandlerFn>([
-        [Handler.Change, handleChange.bind(undefined, propsFn)],
-        [Handler.Click, handleClick.bind(undefined, propsFn)],
-        [Handler.DoubleClick, handleDoubleClick.bind(undefined, propsFn)],
-        [Handler.MouseUp, handleOnMouseUp.bind(undefined, propsFn)],
-        [Handler.Paste, handlePaste.bind(undefined, propsFn)]
+    private readonly handlers = new Map<Handler, HandlerFn>([
+        [Handler.Change, handleChange.bind(undefined, this.propsFn)],
+        [Handler.Click, handleClick.bind(undefined, this.propsFn)],
+        [Handler.DoubleClick, handleDoubleClick.bind(undefined, this.propsFn)],
+        [Handler.MouseUp, handleOnMouseUp.bind(undefined, this.propsFn)],
+        [Handler.Paste, handlePaste.bind(undefined, this.propsFn)]
     ]);
 
-    return (...args: CacheArgs) => {
-        let [
-            handler,
-            rowIndex,
-            columnIndex
-        ] = args;
+    private readonly cache = valueCache<[Handler, number, number]>()((
+        handler: Handler,
+        columnIndex: number,
+        rowIndex: number
+    ) => {
+        let handlerFn = this.handlers.get(handler);
 
-        return cache(args, handlers.get(handler) as HandlerFn, rowIndex, columnIndex);
-    };
-};
+        return handlerFn && handlerFn.bind(undefined, rowIndex, columnIndex);
+    });
```

Comment:
Here the value and the key are generated with the same params and the value is good for the duration of the life of the table. No memoization, just a regular value cache.